gifsicle.yaml

```diff
@@ -0,0 +1,9 @@
+name: gifsicle
+sources:
+  # Source0
+  - type: archive
+    url: http://www.lcdf.org/gifsicle/gifsicle-1.91.tar.gz
+    sha256: 0a4ee602aa244cdcdd86a250a6b39c94d8343cf526b8fae862d8a0efc337a800
+post-install:
```

Comment:
Not needed at all.

Reply:
> Not needed at all.

From the doc:
`Animated GIF files can now be opened frame by frame if you have Gifsicle installed`
`File->Export Animated GIF ... can create an animated GIF file if you have Gifsicle installed`
`Created Gifsicle which is needed for the creation and viewing of animated GIF files`

For example:
```
#define CMD_GIF_CREATE \
        "gifsicle --colors 256 -w -O2 -D 2 -l0 --careful -d %d %s -o \"%s\""
```